src/ts/player-events.d.ts

```diff
@@ -473,11 +473,14 @@ declare namespace bitmovin {
       html?: string;
       region?: string;
       regionStyle?: string;
-      position?: string;
```

Comment:
Hum that seems wrong based on what is described in the issue I linked you, unless there some other case I'm not aware of, could be worth asking @jmsn